app/src/main/java/org/oppia/app/options/AppLanguageActivityPresenter.kt

```diff
@@ -22,7 +24,10 @@ class AppLanguageActivityPresenter @Inject constructor(private val activity: App
       adapter = languageSelectionAdapter
     }
 
-    binding.appLanguageToolbar.setNavigationOnClickListener {
+    val toolbar = activity.findViewById<View>(R.id.app_language_activity_toolbar) as Toolbar
```

Comment:
You need not define toolbar like this. You can get it through binding `activity.setSupportActionBar(binding.appLanguageActivityToolbar)`

Reply:
I have used `binding.appLanguageActivityToolbar` now, but I had to typecast it as ToolBar manually. I cannot use it directly as a ` activity.setSupportActionBar(binding.appLanguageActivityToolbar)` since `binding.appLanguageActivityToolbar` is being taken as of type View! by default.

Samer for others.

Reply:
Ack.